WELCOME.md

```diff
@@ -0,0 +1,47 @@
+👋 Hello @new_user_name and welcome to the For the Horde community! We're so happy you're here! The purpose of this community is to give you a space to connect with other Minnesota-based tech and tech-adjacent folks. Want to be successful here? Give more than you take.
+
+✅ First things first. This is a community and we want this to be a safe and inclusive space for everyone. We expect you to abide by the For the Horde Code of Conduct in this Slack team AND while participating in any in-person or online For the Horde events. Please head on over to [http://git.io/v0d6b](http://git.io/v0d6b).
+
+Need to report Code of Conduct violation? The admins are listed at the bottom of the Code of Conduct. Snap a screenshot, report it.
+
+🚴🏿‍♀️🕺🏻👩‍💻😻⚾️✈️📺 Second, you've automagically been added to the #introductions channel. Please introduce yourself to the community there and share something fun about yourself. Where you're from, what you're working on, what your most peculiar hobby is, what your favorite color is, etc.. We want to get to know you! Also, there's probably a channel for that.
+
+📣 Speaking of channels. We have a lot. When thinking about where to post a message, go through the channel list and check there first. It’s very tempting with ~600 eyeballs to post something to #general. We tend to treat #general like a watercooler, not a place to garner attention.
```

Comment:
```suggestion
📣 Speaking of channels. We have a lot. When thinking about where to post a message, go through the channel list and check there first. It’s very tempting with ~1200 eyeballs to post something to #general. We tend to treat #general like a watercooler, not a place to garner attention.
```
I'm gonna estimate the average number of eyeballs per member to be ~2 😂

Reply:
hahaha yep!